mobile-v3/ios/Runner/AppDelegate.swift

```diff
@@ -11,7 +11,7 @@ import GoogleMaps                                          // Add this import
   ) -> Bool {
     GeneratedPluginRegistrant.register(with: self)
 
-    GMSServices.provideAPIKey("GOOGLE_MAPS_API_KEY")
+    GMSServices.provideAPIKey("AIzaSyAGtjeiEUdsPg5_VLrs1X17nkroKMk5fUk")
```

Comment:
please remove this KEY. revert this change immediately. @jordanopos

Reply:
This file was deleted from the commit! @Baalmart